src/exec/imagemagick.ts

```diff
@@ -1,24 +1,25 @@
 import { sync as commandExists } from 'command-exists';
-import { runSync } from './run.js';
+import { runAsync, runSync } from './run.js';
```

Comment:
is there a reason to have kept some runs synchronous? because thos are short commands that you do no need to run async?

Reply:
lazyness?

those are small commands that are run once on startup, so it's not problematic imho